fr/core-libraries/caching.rst

```diff
@@ -429,16 +438,25 @@ l'API de Cache
 
     Incrémente de manière atomique une valeur stockée dans le moteur de cache.
     Idéal pour modifier un compteur ou des valeurs de sémaphore.
-   
+
 .. php:staticmethod:: decrement($key, $offset = 1, $config = 'default')
 
     Décrémente de manière atomique une valeur stockée dans le moteur de cache.
     Idéal pour modifier un compteur ou des valeurs de sémaphore.
 
+.. php:staticmethod:: add($key, $value, $config = 'default')
+
+    Ajoute des données au cache, mais seulement si la clé n'existe pas déjà.
+    Dans le cas où cette donnée existe, cette méthode va retourner false.
+    Quand les données possibles sont vérifiées et définies de façon atomique.
```

Comment:
Where possible data is checked & set atomically.
-> Lorsque c'est possibles, les données sont vérifiées et définies de façon atomique.